src/pages/patientView/PatientViewPage.tsx

```diff
@@ -448,6 +457,8 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                                     containerWidth={WindowStore.size.width-20}
                                                     sampleIdToMutationGenePanelId={patientViewPageStore.sampleToMutationGenePanelId.result}
                                                     sampleIdToCopyNumberGenePanelId={patientViewPageStore.sampleToDiscreteGenePanelId.result}
+                                                    onSelectGenePanel={patientViewPageStore.toggleModal}
```

Comment:
the modal and its open state are really properties of the view more than the store.  Lets put them on the PagView instead.  Also, there could be more than one modal, so we need to be more specific with the names.